commit fb5f29a0b678ba143ea034eaf861eac43dcac77a (refs/changes/29/229/1)
Author: Johanan Lai <lai.johanan@gmail.com>
Date:   2016-08-05T11:33:33-07:00 (3 years, 2 months ago)
    
    TOOLS-1509 Update contribution guidelines to reflect use of cr.joyent.us

diff --git a/README.md b/README.md
index 69f7be7..afa68a2 100644
--- a/README.md
+++ b/README.md
@@ -10,9 +10,9 @@
 
 # sdc-sapi
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
+This repository is part of the Joyent Triton project (formerly known as SDC).
+For contribution guidelines and general documentation, visit the main
+[Triton](http://github.com/joyent/triton) project page.
 
 SAPI is the Services API.  This API allows operators to configure, deploy, and
 upgrade software using a set of loosely-coupled federated services.
